test/UnitTests/MSTest.CoreAdapter.Unit.Tests/Discovery/TypeEnumeratorTests.cs

```diff
@@ -119,13 +109,12 @@ public void GetTestsShouldReturnBaseTestMethodsFromAnotherAssemblyByDefault()
 
         var tests = typeEnumerator.Enumerate(out _warnings);
 
-        Assert.IsNotNull(tests);
-        Assert.IsTrue(
+        Verify(tests is not null);
+        Verify(
             tests.Any(t => t.TestMethod.Name == "BaseTestMethod"),
             "DummyDerivedFromRemoteTestClass inherits DummyRemoteBaseTestClass from different assembly. BestTestMethod from DummyRemoteBaseTestClass should be discovered by default.");
```

Comment:
Please move this arg to be a comment before the `Verify`. It's currently going in the `expression` param which we don't want to be filled by "user".